ci/azure-pipelines.yml

```diff
@@ -9,3 +9,14 @@ jobs:
         displayName: Build Documentation
       - publish: 'docs/locale/pt_BR/_build/html'
         displayName: Publish Documentation Artifacts
+
+  - job: ja_JP
```

Comment:
Dear satomi, 
If you have no idea bout using `tox`, you do not have to add an Azure pipeline builder section likely. 
This is the root cause of current build failure. 
You could keep this AZP config file unchanged at this moment.

Reply:
Dear Yuxiang,
Thank you very much for your advice!
I understood your point and I will delete this code. We are not going to use tox for now.

Reply:
Sincere apologize for my wrong operations on git commits  

PS: Github desktop is definitely not good tool